src/MathJaxDisplay.js

```diff
@@ -0,0 +1,36 @@
+import React from "react";
+import { MathJax } from "better-react-mathjax";
```

Comment:
ah, you didn't list the lib in the package.json. 
just try `$ npm install --save better-react-mathjax`